Monika After Story/game/zz_music_selector.rpy

```diff
@@ -93,7 +93,9 @@ init -1 python in songs:
             music_choices.append(("Your Reality","bgm/credits.ogg"))
             music_choices.append(("I Still Love You","bgm/monika-end.ogg"))
             music_choices.append(("Okay, Everyone! (Monika)","<loop 4.444>bgm/5_monika.ogg"))
-
+            if seen_event('monika_rain'):
```

Comment:
Prepend `seen_event` with `store` like this: `store.seen_event`. The code here is in the songs `store`, so it doesnt have access to global functions without explicitly calling the `store` or importing. (Do **not** import store here. it is unecessary).

Reply:
Added.

Reply:
alright dude, `seen_event` is still a function, so you need to call it like one as you did before. the only difference is that it needs to have `store` in front of it because of scope. code inside a `store` isnt aware of code outside of its `store` unless you tell it where to find it. (prepending with `store`). Look up scope if you need understanding.